<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Simple CDImage GUI</string>
  </property>
  <property name="whatsThis" >
   <string/>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>541</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7" >
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <property name="margin" >
       <number>15</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblSource" >
          <property name="minimumSize" >
           <size>
            <width>72</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Source Root</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtSource" >
          <property name="whatsThis" >
           <string>Everything in the folder is to be added to the iso image</string>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSource" >
          <property name="maximumSize" >
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QLabel" name="lblTarget" >
          <property name="minimumSize" >
           <size>
            <width>72</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Target File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtTarget" >
          <property name="whatsThis" >
           <string>The location of the generated iso image</string>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnTarget" >
          <property name="maximumSize" >
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QLabel" name="lblVolumLabel" >
          <property name="minimumSize" >
           <size>
            <width>72</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Volume Label</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtLabel" >
          <property name="whatsThis" >
           <string>The volume label of the generated iso image</string>
          </property>
          <property name="text" >
           <string>MyDisc</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" >
        <property name="topMargin" >
         <number>8</number>
        </property>
        <property name="bottomMargin" >
         <number>5</number>
        </property>
        <item>
         <widget class="QGroupBox" name="grpFormat" >
          <property name="minimumSize" >
           <size>
            <width>160</width>
            <height>88</height>
           </size>
          </property>
          <property name="title" >
           <string>Iso Format</string>
          </property>
          <widget class="QWidget" name="layoutWidget" >
           <property name="geometry" >
            <rect>
             <x>20</x>
             <y>20</y>
             <width>75</width>
             <height>65</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2" >
            <item>
             <widget class="QRadioButton" name="rdoIso9660" >
              <property name="text" >
               <string>ISO-9660</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rdoJoliet" >
              <property name="text" >
               <string>Joliet</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rdoUdf" >
              <property name="text" >
               <string>UDF</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" >
            <item>
             <widget class="QCheckBox" name="chkTimeStamp" >
              <property name="text" >
               <string>Reset time stamp</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="dateTimeEdit" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="dateTime" >
               <datetime>
                <hour>0</hour>
                <minute>0</minute>
                <second>0</second>
                <year>2010</year>
                <month>1</month>
                <day>1</day>
               </datetime>
              </property>
              <property name="displayFormat" >
               <string>yyyy/M/d H:mm:ss</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="chkHidden" >
            <property name="text" >
             <string>Include hidden files and folders</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkCompress" >
            <property name="text" >
             <string>Encode duplicate files only once</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkLargeSize" >
            <property name="text" >
             <string>Ignore maximum image size of 681,984,000 bytes</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
        <property name="textVisible" >
         <bool>false</bool>
        </property>
        <property name="invertedAppearance" >
         <bool>false</bool>
        </property>
        <property name="textDirection" >
         <enum>QProgressBar::TopToBottom</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>8</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit" >
        <property name="verticalScrollBarPolicy" >
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
        <property name="acceptRichText" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" >
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnStart" >
          <property name="whatsThis" >
           <string>Start creating the iso image</string>
          </property>
          <property name="text" >
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAbout" >
          <property name="text" >
           <string>About</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnExit" >
          <property name="whatsThis" >
           <string>Exit the application</string>
          </property>
          <property name="text" >
           <string>Exit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_4" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
